windows/shared/logging/PlogTraceLogging/include/PlogTraceLogging/PlogTraceLogging.hxx

```diff
@@ -0,0 +1,46 @@
+
+#ifndef PLOG_TRACE_LOGGING_HXX
+#define PLOG_TRACE_LOGGING_HXX
+
+#include <windows.h>
+
+#include <TraceLoggingProvider.h>
+
+#include <plog/Appenders/IAppender.h>
+#include <plog/Log.h>
+
+#include <string>
+
+TRACELOGGING_DECLARE_PROVIDER(g_hplogTraceLoggingProvider);
+
+namespace plog
+{
+
+template <class Formatter>
```

Comment:
The rest of the codebase is using `typename` instead of `class`, so let's update this to be consistent:
```suggestion
template <typename Formatter>
```

`typename` is more accurate than `class` which is why we prefer it in the codebase (eg. not all template parameter values are classes, but they all are type names).